libsolidity/interface/OptimiserSettings.h

```diff
@@ -146,6 +148,11 @@ struct OptimiserSettings
 	/// them just by setting this to an empty string. Set @a runYulOptimiser to false if you want
 	/// no optimisations.
 	std::string yulOptimiserSteps = DefaultYulOptimiserSteps;
+	/// Sequence of clean up optimisation steps after the above (default or user supplied) set of
```

Comment:
Better to refer to the field by name. The order might change.
```suggestion
	/// Sequence of clean-up optimisation steps to run after yulOptimiserSteps.
```